internal/cli/events/organization/list.go

```diff
@@ -55,8 +54,6 @@ func (opts *ListOpts) Run() error {
 
 	if opts.orgID != "" {
```

Comment:
Once you add `cli.GlobalOpts` to `ListOpts` you should update this piece of code too
```suggestion
	if opts.ValidateOrgID() {
```

Reply:
If you implement this comment https://github.com/mongodb/mongocli/pull/780/files#r681786348 you can cancel the `if`